### DISCUSSION TITLE
Filter Lap time in 2023 test an plot the value on the graph

### DISCUSSION THREAD

**Discussion by conso98** [theOehrly/Fast-F1#322] [Q&A]
Hi, i can't figure it out how to filter the higher time and not plot them, and covert to string and troncate at the minutes on the graph Y=Lap time X=#Lap, actually I'm using this line, but NaT type in some time laps valeu doest allow me to use y.strftime("%M:%S") to troncate te value:

for x,y in zip(third_driver['LapNumber'],third_driver['LapTime']):
    max = datetime.time(0,1,45)
    if(y<max):
        ax.plot(third_driver['LapNumber'], third_driver['LapTime'], color=third_color)
        
    label = str(y)
    plt.annotate(label, (x,y), textcoords="offset points", xytext=(0,10), ha='center')
    
ax.set_xlabel("Time [s]")
ax.set_ylabel("Laps [#]")

**Comment by theOehrly** [comment]
I'm not sure I fully understand your question but

1. Please post a minimal self-contained example when you ask forr help. That is a short snippet of code that one can copy and run without any modifications.

2. You can of course not use ``.strftime`` when a value is NaT. You need to check for NaT values yourself before calling the formatter.
